app/controllers/app.rb

```diff
@@ -39,7 +39,7 @@ class App < Roda
             resource = Github::ResourceMapper
                        .new(MindMap::App.config.GITHUB_TOKEN)
                        .search(search_term, tags)
-
+            puts "resource:#{resource}"    # Mark Here for handling nil case
```

Comment:
Could you remove this? Is this a comment that's supposed to be replaced with functionality?

I'd say the easiest way to do this would be to redirect the user to a 404 page.

```
redirect "/404" unless resource
```

Reply:
Great! I will do it.

Reply:
okay, awesome. Let me know when you add it.

Reply:
I have merged into main with the modification by committing 834de68 and push to main branch.

Reply:
Rather than redirect the page into /404, I add a 'resource_nil' page tell user the  search result is not found case when the resource return nil.

Reply:
Awesome, that's even more useful! :)